app/controllers/application_controller.rb

```diff
@@ -1,2 +1,4 @@
 class ApplicationController < ActionController::Base
-end
+  protect_from_forgery with: :exception
+  include SessionsHelper
```

Comment:
move the include before the `protect_from_forgery` call

Reply:
solved. ✓